### PR TITLE
updated features

### PR DESCRIPTION
**Pull request by priyam1103**
Deployed link [Duskytube](https://duskytube.vercel.app/)

### CODE REVIEW

File: components/UpdateData.js [hunk 1]
```diff
@@ -0,0 +1,41 @@
+import React, { useContext, useEffect } from "react";
+import { GlobalContext } from "../context/GlobalState";
+export default function UpdateData({ data }) {
+  console.log(data);
+  const { updateState } = useContext(GlobalContext);
```
Comment (theshivamverma) [inline components/UpdateData.js:5]: try to make a custom hook in your context and use that to get data from context

File: context/AppReducer.js [hunk 2]
```diff
@@ -0,0 +1,36 @@
+export default (state, action) => {
+  switch (action.type) {
+    case "UPDATE_STATE":
+      console.log(action.payload.dislikes);
```
Comment (theshivamverma) [inline context/AppReducer.js:4]: remove console logs from all places

**Review by theshivamverma — CHANGES_REQUESTED** [review]

**Comment by theshivamverma** [comment]
please provide the deployed url in PR, try using camelCase naming for your variables and functions, it improves code readability.
remove extra spacing, unused code and comments.

**Comment by Akshay-Laxmipathi** [comment]
Add some margin to your pill shaped filter buttons...the "all" button is partly enclosed in the sidebar space

**Pull request merged**